.github/workflows/lint.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.14.x]
+        go-version: [1.16.x, 1.17.x]
```

Comment:
We'll have to keep in mind to upgrade Terraform, Crossplane,etc if they're not already on these versions.

Reply:
Yep, Pipeline will fail.